### ISSUE TITLE
6105 support?

### ISSUE THREAD

**Issue by badass-boss** [ManCloud/UltraCIC-III#3]
Does this code even support 6105/7105?? I can see this snippet in asm code but there is no switch for activating 6105...

.equ CIC_TYPE=0b00    ;6102/7101
;.equ CIC_TYPE=0b01    ;6103/7103
;.equ CIC_TYPE=0b10    ;6106/7106
;.equ CIC_TYPE=0b11    ;6101/7102

**Comment by ManCloud** [comment]
Yes, this is activated by "FLAG_x105_MODE".

It's not really a type but more a patch

**Comment by whipflash191** [comment]
Sorry to necro an old thread but I might have to use this soon myself and am not overly familiar with this type of programing. 

To clarify, if I wanted to program it for a 7105 cic id just need to set the flag to 1? It'll ignore the equ at the start or does it need to be commented out? Id be building this for a Ocarina of Time repair

**Comment by ManCloud** [comment]
yes, if this flag is set to 1 the *105 mode is enabled and therefore an Ocarina of Time module should be compatible.

**Comment by whipflash191** [comment]
Thanks for getting back to me I appreciate it!
